google_ads/application.py

```diff
@@ -203,7 +203,18 @@ def create_google_ads_client(
     }
 
     # Initialize the Google Ads API client with the properly structured dictionary
-    client = GoogleAdsClient.load_from_dict(google_ads_credentials)
+    try:
+        client = GoogleAdsClient.load_from_dict(google_ads_credentials)
+    except Exception:
```

Comment:
Do not delete for all the exceptions. Check the correct exception and handle that alone.